### PR TITLE
add option to use existing ssh key in ec2

### PR DESCRIPTION
**Pull request by itaygvlz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,28 +143,30 @@ terraform apply -var="media_node_ami=ami-08895422b5f3aa64a" -var="media_node_typ
...
+target_subnet_id          = ""
+target_security_group_id  = ""
 target_placement_group_id = ""
-
+ 
 # Media Nodes
-media_node_count = 4
-media_node_type = "is4gen.2xlarge"
+media_node_count = 2
+media_node_type = "i3.2xlarge"
```
Comment (nkorobko) [inline README.md:162]: default should be i4i.2xlarge

File: terraform/aws/examples/ec2/complete/easy_starter/main.tf [hunk 2]
```diff
@@ -19,10 +19,12 @@ locals {
   create_pg  = var.target_placement_group_id == "" && var.avoid_pg == false ? true : false
   deploy_bastion = var.target_subnet_id == "" ? var.deploy_bastion : false
   resource_prefix = "${var.resources_name_prefix}-${random_string.random.result}"
+  create_ssh_key = var.key_name == "" ? true : false
 }
 
 module "ssh_key" {
   source = "../../../../modules/ssh_key"
+  count = local.create_ssh_key ? 1 : 0
```
Comment (nkorobko) [inline terraform/aws/examples/ec2/complete/easy_starter/main.tf:27]: count should be above source

File: terraform/aws/modules/bastion/variables.tf [hunk 3]
```diff
@@ -34,11 +34,6 @@ variable "deployment_tag_name" {
   default = "default-tf-tag"
 }
 
-variable "private_key" {
```
Comment (nkorobko) [inline terraform/aws/modules/bastion/variables.tf:37]: why removed?

File: terraform/aws/modules/nodes/variables.tf [hunk 4]
```diff
@@ -45,11 +45,6 @@ variable "path_to_pem" {
   default = ""
 }
 
-variable "key_value" {
```
Comment (nkorobko) [inline terraform/aws/modules/nodes/variables.tf:48]: why removed?

**Review by nkorobko — CHANGES_REQUESTED** [review]

File: terraform/aws/modules/bastion/variables.tf (hunk 3 above)
Comment (itaygvlz) [reply terraform/aws/modules/bastion/variables.tf:37]: not needed, key pair name is enough

File: terraform/aws/modules/nodes/variables.tf (hunk 4 above)
Comment (itaygvlz) [reply terraform/aws/modules/nodes/variables.tf:48]: not needed, key pair name is enough

**Review by nkorobko — APPROVED** [review]

**Pull request merged**